AX: 7.79
AY: 0.96
AZ: 1.64

BX: 0.92
BY: 0.16
BZ: 1.21

SX: 8.3
SY: 4.085
SZ: 1.945

TZ: 1.72

dist_diff_A_S: -1.053
dist_diff_B_S: -4.998
